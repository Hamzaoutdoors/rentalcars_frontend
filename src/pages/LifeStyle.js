import React from 'react';
import styled from 'styled-components';
import { mobile } from '../responsive';

const Container = styled.div`
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: 100vh;
    width: 100%;
    overflow-x: hidden;
    background-image: url("https://i.ibb.co/H7p4vLV/Car-Showroom-Promotion-Facebook-Post-2.png");
    background-repeat: no-repeat;
    background-size: cover;
    background-position: center;
    padding: 20px;
    ${mobile({
    minHeight: '100vh',
  })};
`;

const ImageContainer = styled.div`
    height: 80vh;
    width: 50vw;
    display: flex;
    flex: 1;
    justify-self: right;
    align-self: center;
    transition: all 0.5s ease;
    position: absolute;
    margin: -50px -30px 0 0;
    background-color: rgba(0, 0, 0, 0.2);
    background-image: url("https://i.ibb.co/j5Hys60/lifestyle.png");
    background-repeat: no-repeat;
    background-size: 100% 100%;
    background-position: center;

    ${mobile({
    width: '80%',
    height: '80%',
    transform: 'translate(-10px, 30px)',
    backgroundImage: 'url("https://i.ibb.co/hBLvjmq/lifestyle-mobile.png")',
  })};
`;

const LifeStyle = () => (
  <Container>
    <ImageContainer />
  </Container>
);

export default LifeStyle;
