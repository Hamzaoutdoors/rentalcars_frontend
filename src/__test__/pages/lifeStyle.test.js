import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import LifeStyle from '../../pages/LifeStyle';
import store from '../../redux/ConfigureStore';

it('render the lifestyle page correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <LifeStyle />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
