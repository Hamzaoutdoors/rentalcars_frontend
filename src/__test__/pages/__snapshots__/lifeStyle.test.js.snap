// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render the lifestyle page correctly 1`] = `
<div
  className="sc-bdvvtL blWCvE"
>
  <div
    className="sc-gsDKAQ fTQIun"
  />
</div>
`;
